### PR TITLE
Adds splash screen

### PR DESCRIPTION
**Pull request by steam**
This adds a fast loading and non-blocking splash screen to cold launches of the app using this article posted by Big Nerd Ranch:

https://www.bignerdranch.com/blog/splash-screens-the-right-way/

The splash does work but further work will be needed to remove the black screen that is displayed while content is loaded into the web view.

### CODE REVIEW

**Comment by jayzes** [comment]
:coffee:

File: app/src/main/res/drawable/background_splash.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (colinta) [inline app/src/main/res/drawable/background_splash.xml]: I'm not familiar with a "layer-list", what is that?

Comment (steam) [inline app/src/main/res/drawable/background_splash.xml]: Its a way of grouping drawables and drawing them in order (had to look that up). http://developer.android.com/guide/topics/resources/drawable-resource.html#LayerList

**Comment by colinta** [comment]
:+1: LGTM!

**Pull request merged**